GameServerLib/Packets/PacketHandlers/HandleSpawn.cs

```diff
@@ -122,13 +122,13 @@ public override bool HandlePacket(int userId, SpawnRequest req)
             {
                 // Shop (blue team)
                  _game.PacketNotifier.NotifyStaticObjectSpawn(userId, 0xff10c6db);
-                 _game.PacketNotifier.NotifySetHealth(userId, 0xff10c6db);
+                _game.PacketNotifier.NotifyEnterLocalVisibilityClient(userId, 0xff10c6db);
```

Comment:
`0xff10c6db` is some hardcoded `NetId`. Add comment: `// TODO: Generate shop NetId to avoid hard-coding`